.mergify.yml

```diff
@@ -0,0 +1,8 @@
+pull_request_rules:
+  - name: backport
+    conditions:
+      - label=stable backport potential
+    actions:
+      backport:
+        branches:
+          - stable/0.4
```

Comment:
Is there some way we can automate the version here so we don't have to mess with it?

We use a python package in the docs config [here](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/105821a9482bea60dd79d64dea7d786cc7e59903/docs/conf.py#L33). I don't have any ideas ready for this yml config file, just thought I'd ask

Reply:
I think no matter what, it's going to need to be hard-coded here and updated.  That's just the nature of yaml.

In principle, though, we might be able to _automate_ the bumping of version numbers using a tool like Commitizen.  But I'm not sure how it would deal with the patchlevel only being needed in certain locations, e.g. some places we want `0.4.0` and others (such as here) we just want `0.4`.